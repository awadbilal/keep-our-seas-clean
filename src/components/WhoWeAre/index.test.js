import React from "react";
import renderer from "react-test-renderer";
import WhoWeAre from "./index";

it("renders correctly", () => {
  const tree = renderer.create(<WhoWeAre />).toJSON();
  expect(tree).toMatchSnapshot();
});
