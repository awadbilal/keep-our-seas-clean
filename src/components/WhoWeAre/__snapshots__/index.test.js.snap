// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt-5 mb-5 container"
>
  <div
    className="whoWeAre row"
  >
    <div
      className="col-md-8 offset-md-2 text-center col"
    >
      <h2
        className="titles"
      >
        whoWeAre.title
      </h2>
      <p
        className="mt-4"
      >
        contact.team.paragraph
      </p>
    </div>
  </div>
</div>
`;
